kodi_addon_checker/check_addon.py

```diff
@@ -86,6 +86,9 @@ def start(addon_path, args, all_repo_addons, config=None):
             if KodiVersion(args.branch) >= KodiVersion("isengard"):
                 check_files.check_for_legacy_language_path(addon_report, addon_path)
 
+            if KodiVersion(args.branch) >= KodiVersion("leia"):
```

Comment:
Should be krypton: https://kodi.wiki/view/Add-on_structure#changelog.txt

Reply:
Indeed, I wasn't sure. But I will change that.